backend/app/services/ml_engine.py

```
@@ -0,0 +1,376 @@
+        "восходящая аорта нисходящая аорта дуга аорты перешеек "
+        "максимальный диаметр минимальный диаметр периметр площадь поперечного сечения "
+        "мм см таблица рекомендации"
+    )
+    return f"{base_terms} {' '.join(facts)}".strip()
+
+@track_node_time
+def ingest_request(state: MedGraphState) -> MedGraphState:
+    errors = list(state.get("errors", []))
+    warnings = list(state.get("warnings", []))
+    if not state.get("query", "").strip():
+        errors.append("Не передан query")
```

Ruff (0.15.12)

[warning] String contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF001)